src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -224,8 +224,11 @@ public void visit(NodeTraversal t, Node n, Node parent) {
           Node callNode = node;
           String requiredNamespace = callNode.getLastChild().getString();
           String localName = n.getFirstChild().getQualifiedName();
-          convertRequireToImportStatements(
-              n, Collections.singletonList(localName), requiredNamespace, false);
+          ModuleImport moduleImport =
```

Comment:
I feel most of the body of this 'visit' method should be in the ctor for ModuleImport, or at least a static method (you might want to return null if the node is misidentified).

Reply:
I could move the entire body of `visit()` to a `maybeConvertRequireToImport(Node n)` function but not sure what we get from that. The current implementation seems quite readable to me.